samples/DiscordBindingSample/DiscordBinding.cs

```diff
@@ -1,4 +1,4 @@
-using Azure.Storage.Queues;
+﻿using Azure.Storage.Queues;
```

Comment:
I have no idea what sort of whitespace is here that GitHub "hide whitespace changes" still keeps presenting this diff. Go figure :)